src/pykeen/evaluation/sklearn.py

```diff
@@ -19,46 +19,37 @@
     "SklearnMetricResults",
 ]
 
+_fields = [
+    (
+        metadata.func.__name__,
+        float,
+        field(
+            metadata=dict(
+                name=metadata.name,
+                doc=metadata.description or "",
+                link=metadata.link,
+                range=metadata.interval(),
+                increasing=metadata.higher_is_better,
+                f=metadata.func,
+            )
+        ),
+    )
+    for metadata in classifier_annotator.metrics.values()
+]
+
+SklearnMetricResultsBase = make_dataclass(
```

Comment:
Should we put the `rexmex` metrics into a different evaluator? Or rename `SklearnMetricEvaluator`?

Reply:
we should definitely rename this evaluator! I just didn't want to shake up even more stuff in this PR, but I'll do that